Packages/Tracking/Core/Runtime/Scripts/ActionDetectors/GrabDetector.cs

```diff
@@ -34,9 +34,44 @@ public class GrabDetector : ActionDetector
         public bool IsGrabbing => IsDoingAction;
 
         // Accessor actions for readability
-        public Action<Hand> OnGrabStart => onActionStart;
-        public Action<Hand> OnGrabEnd => onActionEnd;
-        public Action<Hand> OnGrabbing => onAction;
+        public Action<Hand> OnGrabStart
+        {
+            get
+            {
+                return onActionStart;
+            }
+
+            set
+            {
+                onActionStart = value;
+            }
+        }
+
+        public Action<Hand> OnGrabEnd
+        {
+            get
+            {
+                return onActionEnd;
+            }
+
+            set
+            {
+                onActionEnd = value;
+            }
+        }
+
+        public Action<Hand> OnGrabbing
+        {
+            get
+            {
+                return onAction;
+            }
+
+            set
+            {
+                onAction = value;
+            }
+        }
```

Comment:
You could also write this as expression bodied properties with setters to retain the more concise syntax:

```suggestion
        public Action<Hand> OnGrabStart
        {
            get => onActionStart;
            set => onActionStart = value;
        }

        public Action<Hand> OnGrabEnd
        {
            get => onActionEnd;
            set => onActionEnd = value;
        }

        public Action<Hand> OnGrabbing
        {
            get => onAction;
            set => onAction = value;
        }
```